lib/types/package.json

```diff
@@ -20,6 +20,8 @@
     "@types/node": "^20.8.2",
     "nodemon": "^3.0.1",
     "ts-node": "^10.9.1",
-    "typescript": "^5.2.2"
-  }
+    "typescript": "^5.2.2",
+    "bun-types": "latest"
```

Comment:
Don't you think we should stuck with one runtime ( NodeJs )? Adding bun to the project may cause conflict with NodeJs-related things.

Reply:
It will also add confusion to the new comer